deb/openmediavault/workbench/proxy.conf.json

```diff
@@ -13,5 +13,10 @@
     "target": "http://omv6box.local/",
     "secure": false,
     "logLevel": "debug"
+  },
```

Comment:
I added this, so that it can get the navigation info etc from vagrant when running locally